### DISCUSSION TITLE
blog/2-tahun-dan-masih-berlanjut

### DISCUSSION THREAD

**Discussion by giscus** [hendraaagil/website#51] [Comments]
# blog/2-tahun-dan-masih-berlanjut

Cerita singkat tentang karir saya selama 2 tahun ke belakang.

https://hendraaagil.dev/blog/2-tahun-dan-masih-berlanjut

**Comment by kholishafid** [comment]
ceritanya mantap mas, jadi motivasi :thumbsup:

**Comment by wiratmo** [comment]
Gil, ijin share ke adik adikmu ya

**Reply by hendraaagil** [reply]
waduh wkwkw, siap pak 🤣

**Comment by teguhlerenglawu78** [comment]
Terimakasih mas

Panjenengan luar biasa ... Semoga barokah hidupnya

**Reply by hendraaagil** [reply]
aamiin, makasih pak 🙏
